_query-dsl/specialized/script-score.md

```diff
@@ -0,0 +1,354 @@
+---
+layout: default
+title: Script score
+parent: Specialized queries
+grand_parent: Query DSL
+nav_order: 60
+---
+
+# Script score query
+
+A `script_score` query lets you customize the score calculation by using a script. For an expensive scoring function, you can use a `script_score` query to calculate the score only for the returned documents that have been filtered.
+
+## Example
+
+For example, the following request creates an index with one document:
+
+```json
+PUT testindex1/_doc/1
+{
+  "name": "John Doe",
+  "multiplier": 0.5
+}
+```
+{% include copy-curl.html %}
+
+You can use a `match` query to return all documents that match `John`:
+
+```json
+GET testindex1/_search
+{
+  "query": {
+    "match": {
+      "name": "John"
+    }
+  }
+}
+```
+{% include copy-curl.html %}
```

Comment:
this is shown up on doc preview, should this be a dynamic html content?

Reply:
This does not render in GitHub-flavored markdown but does in local build.